ntc_templates/templates/cisco_ios_ping_df.textfsm

```diff
@@ -0,0 +1,22 @@
+Value Required SENT_QTY (\d+)
```

Comment:
it looks like this is a duplicate of the above template

Reply:
thank you, that was a duplicate, i was trying to test the DF function as part of a ping and forgot about this... i will remove